index.bs

```diff
@@ -1687,6 +1687,31 @@ Open Screen Protocol messages.  An agent should not send extension fields to
 another agent unless that agent advertises an extension capability ID in its
 [=agent-info=] that indicates that it understands the extension fields.
 
+Type Key Backwards Compatibility
+--------------------------------
+
+As messages are modified or extended over time, certain rules must be followed
+to maintain backwards compatibiilty with agents that understand older versions
+of messages.
```

Comment:
I'm assuming these rules both apply to top-level messages, and nested message definitions like `remote-playback-state` and `agent-info`.  Maybe call that out specifically?

Reply:
Yes, I made it explicit.